toc.json

```diff
@@ -332,6 +332,15 @@
       "title": "SCIM APIs",
       "uri": "openapi/scim/scim.yaml"
     },
+    {
+      "type": "divider",
+      "title": "CRM"
+    },
+    {
+      "type": "item",
+      "title": "CRM APIs",
+      "uri": "openapi/crm/crm.json"
+    },
```

Comment:
I know it was in the AC of the user story but I wonder if we should wait to add the CRM section to the public docs until we have a few more of the endpoints figured out.

Reply:
I have mark these endpoints as proposed.